client/utils/evalAuth.js

```diff
@@ -0,0 +1,15 @@
+export const evalAuth = (data = {}) => {
```

Comment:
Super optional: maybe including "eval" in the name will make people think about the `eval` function in JS. Maybe some other word like "check" would work better?

Reply:
Well it's also not really a "check" I'd say. That indicates a boolean to me. It is forming (evaluating?) a simplified auth object to be returned.

> maybe including "eval" in the name will make people think about the eval function in JS

I do understand that though. `evaluateAuth` maybe? Along with adding the missing JSDocs.

Reply:
Sure! That would be good.